specification/06-arch_overview.adoc

```diff
@@ -43,7 +44,9 @@ The device attestation process is specific to each TVM workload configuration,
 but it may be handled by the Trusted Device Manager (TDM).
 
 In order to protect the confidentiality and integrity of TVM data between the
-PCIe root port and a bound TDI, CoVE-IO relies on the TSM to configure both the
-PCIe physical device and the root port it's attached to in order to establish
-IDE streams between the two. The PCIe root port IDE configuration from the TSM
-is abstracted through the platform Root-of-Trust (RoT).
+PCIe root port and a bound TDI, PCIe IDE streams are established between them.
+CoVE-IO relies on the TSM to configure the IDE keys and capabilities for both
```

Comment:
> TSM to configure the IDE keys and capabilities

I think TSM can only configure the IDE keys, but not it cannot configure the capabilities.

Reply:
Fixed.